sigs/observability.md

```diff
@@ -183,20 +187,24 @@ Examples include:
 
 ## Operations
 
-- Formation of the SIG follows the [documented process][sigform].
-- [Roles][sigroles] for SIG Observability
-  - TOC Liaison: *Jeff Brewer*\*
-  - SIG Chairs: [Matt Young](https://github.com/halcyondude), [Richard Hartmann](https://github.com/RichiH)
-  - Tech Leads: [Michael Hausenblas](https://github.com/mhausenblas), [Bartłomiej Płotka](https://github.com/bwplotka), *Richard Hartmann*\*
+- The formation of the SIG follows the [documented process][sigform].
 
-\*_**(TODO: need confirmation)**_
+- [Roles][sigroles] for SIG Observability
+  - TOC Liaison: [Brendan Burns][Brendan Burns]
+  - SIG Chairs:
+    - [Matthew Young][matthew young]
+    - [Richard Hartmann][Richard Hartmann]
+    - TBD
+  - Tech Leads ([sigtlprocess])
+    - Bartłomiej Płotka
 
 [sigform]: https://github.com/cncf/toc/tree/master/sigs#sig-formation-process
 [sigroles]: https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-member-roles
+[sigtlprocess]: https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections
 
 - Contact
   - Slack channel: #sig-observability @ [https://cloud-native.slack.com](https://cloud-native.slack.com)
   - Email List: [cncf-sig-observability@lists.cncf.io](mailto:cncf-sig-observability@lists.cncf.io)
 - Meeting Schedule:
   - TBD - pending feedback from SIG members
-  - [https://www.cncf.io/community/calendar](https://www.cncf.io/community/calendar/)
+  - [https://www.cncf.io/community/calendar](https://www.cncf.io
```

Comment:
This looks like a link that won't resolve.

Reply:
fixed

Reply:
Still off?